crypto/vrf/vrf_libsodium.go

```diff
@@ -18,6 +18,8 @@ func init() {
 	defaultVrf = newVrfEd25519libsodium()
 }
 
+const ProofSize = int(libsodium.PROOFBYTES)
```

Comment:
I want to know bytes value. How about adding following comments?
```go
const ProofSize = int(libsodium.PROOFBYTES) // 80 bytes
```

Reply:
ditto